### DISCUSSION TITLE
how I can fill the missing line in table so I can extract the whole table info

### DISCUSSION THREAD

**Discussion by hquzhuguofeng** [jsvine/pdfplumber#639] [Ask for help with specific PDFs]
![image](https://user-images.githubusercontent.com/45601044/162363547-1b693780-c1f3-4531-987b-d68ca6c307e6.png)

when extract_tables() I missing the last col, so how can I fill the missing table line then I can extract the whole table

**Comment by hquzhuguofeng** [comment]
I solve this questions, thanks, YYDS

**Reply by manish291740** [reply]
> I solve this questions, thanks, YYDS

What is a Solutions?
